src/mods/07-system-tools-install-mod/install.sh

```diff
@@ -94,12 +94,21 @@ apt install $INTERACTIVE \
     gettext \
     cracklib-runtime \
     libfuse2t64 \
-    libfuse3-4 \
     libopengl0 \
     initramfs-tools \
     --no-install-recommends
 judge "Install basic system tool packages"
 
+print_ok "Installing libfuse for specific release..."
+case $TARGET_UBUNTU_VERSION in
+    "questing")
```

Comment:
I think it's not needed because 1.4 branch is for questing while 1.3 is for plucky. But nice to have.